.ocamlformat

```diff
@@ -1,5 +1,4 @@
 profile=ocamlformat
-version=0.14.1
```

Comment:
I wondered if instead this should be bumped, but we don't specify a version in xen-api#master

Reply:
using the version introduces churn, seeing that our opam repo has only one version I think the versions we use should be synchronized most of the time.